blint/analysis.py

```diff
@@ -17,7 +17,7 @@
 
 from blint.binary import parse
 from blint.logger import LOG, console
-from blint.utils import find_exe_files, is_fuzzable_name, parse_pe_manifest
+from blint.utils import find_exe_files, is_fuzzable_name, parse_pe_manifest, is_ignored_file, is_exe
```

Comment:
May be run isort to sort these import statements?

Reply:
Done.